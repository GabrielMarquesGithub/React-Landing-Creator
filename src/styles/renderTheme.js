import { render } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import { theme } from './theme';

export const renderTheme = (element) => {
  return render(<ThemeProvider theme={theme}>{element}</ThemeProvider>);
};
