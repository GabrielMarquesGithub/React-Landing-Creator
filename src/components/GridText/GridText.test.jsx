import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/renderTheme';
import { GridText } from '.';

describe('<GridText />', () => {
  it('should render', () => {
    renderTheme(<GridText>Children</GridText>);
    expect(screen.getByRole('heading')).toBeInTheDocument();
  });
});
