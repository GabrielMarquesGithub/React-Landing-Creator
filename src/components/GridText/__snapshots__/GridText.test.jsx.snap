// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridText /> should render 1`] = `
<div>
  <div
    class="sc-dkzDqf jqUFFT"
  >
    <section
      class="sc-gsnTZi jiuFls"
    >
      <div
        class="sc-eCYdqJ fLTjpd"
      >
        <h2
          class="sc-bczRLJ exNOxm"
        >
          My grid
        </h2>
        <p
          class="sc-hKMtZM SUpSD"
        >
          Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
        </p>
        <div
          class="sc-jSMfEi cjPvZi"
        >
          <div
            class="sc-gKXOVf bucRnE"
          >
            <h3
              class="sc-bczRLJ jEdyJu"
            >
              Teste 1
            </h3>
            <p
              class="sc-hKMtZM SUpSD"
            >
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
            </p>
          </div>
          <div
            class="sc-gKXOVf bucRnE"
          >
            <h3
              class="sc-bczRLJ jEdyJu"
            >
              Teste 2
            </h3>
            <p
              class="sc-hKMtZM SUpSD"
            >
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
            </p>
          </div>
          <div
            class="sc-gKXOVf bucRnE"
          >
            <h3
              class="sc-bczRLJ jEdyJu"
            >
              Teste 3
            </h3>
            <p
              class="sc-hKMtZM SUpSD"
            >
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
            </p>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;
